src/zerocash/Zerocash.h

```diff
@@ -33,6 +33,7 @@
 #define ZC_V_SIZE          8
 #define ZC_RHO_SIZE        32
 #define ZC_R_SIZE          48
+#define ZC_MEMO_SIZE       128
```

Comment:
I'm not sure; we don't even use it. Almost all of the code you're looking at is being removed anyway.